compatibility/src/membership.rs

```diff
@@ -0,0 +1,48 @@
+use std::collections::BTreeSet;
```

Comment:
Something like this would work:
```rs
// Added by [ppamorim](https://github.com/ppamorim)
// Taken from https://github.com/datafuselabs/openraft/blob/209ae677ade5b624fea9f6630e9ff191963f5d74/openraft/src/membership/membership.rs#L21
// License: Openraft is licensed under the terms of the MIT License or the Apache License 2.0, at your choosing.
```

Reply:
Thank you